src/parse/ida.py

```
@@ -0,0 +1,38 @@
+import pandas as pd
+import numpy as np
+
+from src.parse.masstable import parseFLASHDeconvOutput, getMSSignalDF, getSpectraTableDF
+from src.render.compression import downsample_heatmap, compute_compression_levels
+from scipy.stats import gaussian_kde
+
+def parseIda(
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)